core/src/main/scala/algebra/instances/StaticMethods.scala

```diff
@@ -113,6 +113,51 @@ object StaticMethods {
     }
   }
 
+  def gcd(a: BigDecimal, b: BigDecimal): BigDecimal = {
```

Comment:
Ah - glad to see my GCD implementations live on :) I think I had problems writing property tests for this (was getting some exception or the other), but didn't figure out the problem. May be worth creating an issue for.